api-reference/10 UI Components/dxNumberBox/1 Configuration/format.md

```diff
@@ -8,13 +8,115 @@ default: ''
 Specifies the value's display format and controls user input accordingly.
 
 ---
-[note] With this property specified, a press on Minus Sign (-) inverts the current value instead of entering "-".
-
-[note] If you set this property, the telephone keyboard is used for editing on mobile devices. However, it may not have a point, comma, or other symbols for entering decimals. Set the [mode](/api-reference/10%20UI%20Components/dxNumberBox/1%20Configuration/mode.md '/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#mode') property to *"text"* to use the standard keyboard instead.
-
 #include common-demobutton with {
     url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/NumberBox/Formatting/"
 }
 
+The format property can accept the following value types:
+
+- Predefined format    
+
+        <!-- tab: JavaScript -->
+        const number = 1.2345;
+
+        type: "fixedPoint" // 1
+        type: "decimal" // 1.2345
+        type: "percent" // 123%
+        type: "currency" // $1
+
+    You can specify the [precision](/Documentation/ApiReference/Common/Object_Structures/format/#precision) property for decimal numbers.
+
+    NumberBox does not support large number formats. If you want to apply these formats, you can implement the [formatter](/api-reference/50%20Common/Object%20Structures/format/formatter.md '/Documentation/ApiReference/Common/Object_Structures/format/#formatter') and [parser](/api-reference/50%20Common/Object%20Structures/format/parser.md '/Documentation/ApiReference/Common/Object_Structures/format/#parser') functions or specify a custom format string.
```

Comment:
There is a global type [format](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type), which supports small and large numbers. I really didn't know how to adequately mention the fact that NumberBox does not support large number format types without showing the types in this description. I think I don't need to show them here, since customers can't use them anyway. If I add a link, e.g. [large number formats](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type), would it be enough?

Reply:
One way to deal with it, is simply not mention those formats. I don't expect a lot of people to care about those formats. We draw attention to the fact that something may not be supported. 

Or maybe phrase is in a bit more positive way - turn the order around. (Don't say "something is not supported". Everything is supported, you just need to know how to enable it :-) ) And then you better add a link, yes. 

```suggestion
    If you want to apply a different format, you can implement the [formatter](/api-reference/50%20Common/Object%20Structures/format/formatter.md '/Documentation/ApiReference/Common/Object_Structures/format/#formatter') and [parser](/api-reference/50%20Common/Object%20Structures/format/parser.md '/Documentation/ApiReference/Common/Object_Structures/format/#parser') functions or specify a custom format string. For example, you can use this functionality to enable [large number formats](/Documentation/ApiReference/Common/Object_Structures/format/#type).
```